docs/06-concepts/01-working-with-endpoints.md

```diff
@@ -77,12 +77,12 @@ The return type must be a typed Future. Supported return types are the same as f
 
 ### Ignore an entire `Endpoint` class
```

Comment:
I would leave the headings as "Ignore …" instead of "Do not generate code for an entire `Endpoint` class" etc. as that, to me, succinctly expresses the outcome.